### ISSUE TITLE
The 2020 CF meeting: Online, 9-11 June

### ISSUE THREAD

**Issue by davidhassell** [cf-convention/discuss#53]
Hello,

The CF meeting on 9-11 June is now less than two weeks away and the registration form and agenda are ready:

* [Registration](https://forms.gle/RoKHT4h8hnBN23Pk7)
* [Agenda](http://cfconventions.org/Meetings/2020-Workshop.html)

Registration is only to get an idea of numbers and so that everyone has an idea of who is attending the meeting. There is, of course, no fee! The registration form also asks you to express an interest in the Breakouts for Current Discussion Topics sessions. Please select as many of these are of interest, even though the discussions are happening in parallel, as it will useful to have a rough idea of numbers for each session.

The registration form also contains a link to a [document](https://docs.google.com/document/d/1urPWngzDCuHTrfpA8nedGoRDVKXs5OmjqO8M6i3UZJM) that contains a very short summary of each discussion topic to help you decide which one, or ones, you might like to discuss. Many thanks to everyone who compiled it at short notice.

We will be using Zoom for the meeting (connection details nearer the time), which will allow each of our parallel discussions to be in its own breakout "room". These rooms can be accessed and switched between without having to leave the main meeting.

We look forward to seeing and hearing from you soon, all the best,

David, on behalf of the organising committee

**Comment by JonathanGregory** [comment]
May I ask, what is the discussion item about standard names going to be concerned with?

**Comment by davidhassell** [comment]
@JonathanGregory There is currently no specific agenda for that discussion topic, other than the knowledge that many people (30/41 of those who have registered so far) have expressed an interest in discussing _something_ about standard names. I think that we envisaged that discussion would flow freely following on from Alison's presentation of the previous day.

However, if people have specific topics they would like to be talked about - we can add them to the discussion topic summary page, which now has summaries of all of the other issues up for discussion: https://docs.google.com/document/d/1urPWngzDCuHTrfpA8nedGoRDVKXs5OmjqO8M6i3UZJM

**Comment by shaheen19** [comment]
Hi David, Is this workshop for advanced users? I have Ph.D in atmospheric physics and chemistry. I started taking evening classes since 2018 to learn python programming and GIS to study atmosphere-biosphere-hydrosphere interactions. I am really glad that I took this route and learning a lot about netCDF files but I am not sure if the participants are going to discuss the most efficient data processing or archiving. It would be great to know some folks studying earth's various processes related to climate change.  Any suggestion would be greatly appreciated. 
Ruby

**Comment by davidhassell** [comment]
Hi Ruby,

The workshop is very much for everyone, whatever their experience.

There is usually a fair amount of background material in the plenary sessions, so that everyone knows what's going on. A breakout discussion topic may pick up on an already advanced discussion, but they all will be preceded by an introduction and overview to bring people who are new to that particular issue up to speed.

I hope that helps, and that you can make the meeting,
David

**Comment by davidhassell** [comment]
Hello,

Just a reminder to register for the CF meeting on Tuesday to Thursday next week, if you have not already done so, by following the link from the [meeting agenda](http://cfconventions.org/Meetings/2020-Workshop.html). If you want to attend, it's important to register so that information about the meeting can be sent out via e-mail.

Many thanks,
David

**Comment by sjskhalsa** [comment]
I haven't received connection information for the meeting tomorrow. Now beginning to doubt that I registered (didn't get an confirmation, or my spamblocker trapped it)
Pls post connection info here

**Comment by davidhassell** [comment]
Hello,

Apologies for the delay - the meeting will be help on Zoom, and the connection details will be added to the [meeting agenda](http://cfconventions.org/Meetings/2020-Workshop.html) later today.

@sjskhalsa - you have indeed registered (I don't think the form we used sent out confirmations ...)

**Comment by sjskhalsa** [comment]
> Hello,
> 
> Apologies for the delay - the meeting will be help on Zoom, and the connection details will be added to the [meeting agenda](http://cfconventions.org/Meetings/2020-Workshop.html) later today.
> 
> @sjskhalsa - you have indeed registered (I don't think the form we used sent out confirmations ...)

Thanks @davidhassell, much appreciated

**Comment by taylor13** [comment]
I must be missing something.  I think it is now the day of the meeting in the meeting's time zone, but I don't see the zoom connection information on the meeting agenda.  I plan to connect in about 13 hours, but I'm still not sure how to do that.  I'll look for some link in the morning.
Thanks, and apologies if I'm missing something obvious.

**Comment by erget** [comment]
Hello everybody,

The connection info is now available in each of the "Questions/Discussion" Google Docs linked next to each session's heading in [the meeting agenda](http://cfconventions.org/Meetings/2020-Workshop.html). For convenience this is link will take you to the opening plenary session:
https://eumetsat.zoom.us/j/9636813552?pwd=VTFudW5iQ0M2VWFRSFpFV215YUZ4QT09

Looking forward to a great workshop starting in 12 hours!
Best regards,
Daniel

**Comment by davidhassell** [comment]
Hello,

The organising committee would very much like thank to everyone for turning up in record numbers and making the meeting such a success.

The google drive with the presentations and notes will be remain where it is, and some of that content will soon make it's way into the [Meetings section](http://cfconventions.org/Meetings/) of the CF website.

See you next year, somewhere, and hopefully somewhere on GitHub much sooner.

**Comment by erget** [comment]
Dear community,

The CF meeting that took place from 9-11 June 2020 has concluded and the organising committee has finished archiving the material from that meeting. If you'd like to find a summary of the workshop, the individual sessions, the chat archive, or any of the presentations or papers that were used during the meeting, please [visit the meeting page](http://cfconventions.org/Meetings/2020-Workshop.html).

The work goes on!

The outputs of sessions related to individual issues are being tracked in those issues; outputs related to proposals from the community at large concerning CF as a whole or its governance are tracked on a [dedicated project board](https://github.com/orgs/cf-convention/projects/1). This will allow us to track how the contributions from you, the community, are followed up moving forward.

Thank you all for an excellent workshop and the work you've put in thus far; we're looking forward to continuing to advance CF with you! See you on GitHub.

Best regards,
The organising committee
